ideas/README.md

```diff
@@ -0,0 +1,141 @@
+# Ideas
+
+## Intro
+
+Welcome to the Google Summer of Code ideas repository!
+Here you can propose new ideas to participate to this fantastic event 
+and contribute to the nix/nixOS/nixpkgs community ecosystem!
+
+Below you will find all instructions to propose your ideas, so don't be shy
+to contribute!
+
+Summerily yours,
+the repo maintainers.
+
+## Submission fields explanation
+
+### ID
+The ID is composed by the current year and an incremental 4 digit integer
+assigned by one of the repo maintainers and will also be the directory name
+that will contain all the artifacts for that idea.
+
+### Title
+The title should be clear, concise and inspirative.
+
+It should also be prefixed by a categorey chosen among:
+    * nix
+    * nixpkgs
+    * nixos
+Depending on which area of the ecosystem more closely fits your idea.
+
+### Author
+Just the author's github handle.
+
+### Proposed Menthor
+If you're in contact with a member of the nix community who's interested in
+helping you as your direct menthor, you can mention the github handle here so
+that the repo maintainers wil be added to the conversation in the PR.
+
+### Effort
+This is the "size" of the effort level of the idea. It should be one of:
+    * SMALL
+    * MEDIUM
+    * LARGE
+
+### Difficulty
+This is how "hard" it is at a technical level to implement the idea. It can be:
+    * EASY
+    * MEDIUM
+    * HARD
+### Overview/Abstract
+This is a short description of the idea. Try to stay within the limit of 400
+characters. Think of this as your "elevator pitch".
+### Description
+Here you can add all the nitty gritty details of the idea. Feel free to add
+anything that you think is relevant for the staff to understand and evaluate
+your idea.
+### Linkography
+Just the link to the relevant documentation and/or description you wrote
+above.
+
+## Idea Template
+
+This would be a PR aimed at creating a README.md file with the content below
+in the directory [ideas/2024/0000](./ideas/2024/0000).
+
+### Metadata
+
+**ID**: [2024/0000](./ideas/2024/0000)
+
+**Title**: [nixpkgs] My idea to nixify the world!
+
+**Author**: @your-github-handle-here
+
+**Proposed Menthor**: [optional] @your-menthor-github-handle-here
+
+**Effort**: SMALL/MEDIUM/LARGE
+
+**Difficulty**: EASY/MEDIUM/HARD
+
+**Overview/Abstract**: Description in less than 400 characters (5 lines, 80 chars).
+
+### Description
+
+Detailed description of the project.
+
+### Linkography
+
+* [Link1](https://localhost/THIS_IS_CLEARLY_A_BROKEN_LINK): Link1 explanation
+* [Link2](https://localhost/THIS_IS_CLEARLY_A_BROKEN_LINK): Link2 explanation
+* [Link3](https://localhost/THIS_IS_CLEARLY_A_BROKEN_LINK): Link3 explanation
```

Comment:
what is the usecase for "linkography"?

Reply:
It's used in case you have links that should to be available at a glance as external reference for the reader (say the project homepage, docs, etc..).